tests/dataframe/test_show.py

```diff
@@ -24,48 +24,3 @@ def test_show_some(make_df, valid_data, data_source):
     elif variant == "arrow":
         assert df_display.preview.dataframe_num_rows == len(valid_data)
     assert df_display.num_rows == 1
-
-
-def test_show_from_cached_collect(make_df, valid_data):
```

Comment:
concept of 'cached_collect' is not correct with this PR so i removed these tests

Reply:
I guess "cached" would be if we called `__repr__()` twice in a row now?